NEWS.rst

```diff
@@ -16,6 +16,8 @@ tested on PyPy3.6.1 v7.1.1-beta0.
 
 **We have dropped support for Python 2 now.**
 
+Use iter instead of getiterator for Python 3.9 compatibility.
```

Comment:
This needs context to be meaningful - you need to say "in ElementTree" or something like that. Overall given all the general Python 2/3 work I don't think this needs to be in the NEWS file since this isn't something our users will see directly.

Reply:
I have removed the entry.

Reply:
Thanks!